### PR TITLE
feat(fast-usdc): add cli config and args for deposit and withdraw

### PR DESCRIPTION
**Pull request by samsiegart**
refs https://github.com/Agoric/agoric-sdk/issues/10328

### CODE REVIEW

File: packages/fast-usdc/test/cli/cli.test.ts [hunk 1]
```diff
@@ -105,19 +105,115 @@ test('shows help for config show command', async t => {
...
+});
+
+test('shows error when deposit command is run with invalid fee', async t => {
+  const output = await collectStdErr([
+    CLI_PATH,
+    'deposit',
+    '50',
+    '--fee',
+    'not-a-number',
+  ]);
```
Comment (dckc) [inline packages/fast-usdc/test/cli/cli.test.ts:139]: To make these look more like command lines, consider...

```suggestion
  const output = await collectStdErr([CLI_PATH, ...'deposit 50 --fee not-a-number'.split(' ')]);
```

**Review by dckc — APPROVED** [review]
looks fine.

All these tests make me curios... do we have code coverage numbers? Were you aiming for complete coverage or something?

**Comment by samsiegart** [comment]
>All these tests make me curios... do we have code coverage numbers? Were you aiming for complete coverage or something?

We do have codecov if you look in the workflows: https://github.com/Agoric/agoric-sdk/pull/10487/checks?check_run_id=33023398762

I think testing as close to 100% of the desired behavior as possible is the goal. Sometimes it's way more effort than reasonable, but these tests are pretty easy to add, so no justification for not doing it. I'm not sure where else we'd set the bar.

File: packages/fast-usdc/test/cli/cli.test.ts (hunk 1 above)
Comment (samsiegart) [reply packages/fast-usdc/test/cli/cli.test.ts:139]: Not really my preference, though I see where you're coming from.

**Comment by dckc** [comment]
> I'm not sure where else we'd set the bar.

I only see tests of failure conditions here. I'm used to at least 1 happy-path-test.

**Pull request merged**